dashboard/app/models/script.rb

```diff
@@ -658,7 +658,7 @@ def self.add_script(options, raw_stages)
       end
 
       stage_name = raw_script_level.delete(:stage)
-      properties = raw_script_level.delete(:properties)
+      properties = raw_script_level.delete(:properties) || {}
```

Comment:
presumably the `if properties` below can then be removed, since `properties` will always be defined